procedures/foreman/branch.md.erb

```diff
@@ -69,32 +71,56 @@
   * [ ] [foreman-selinux](https://github.com/theforeman/foreman-selinux)
   * [ ] [smart-proxy](https://github.com/theforeman/smart-proxy)
 * [ ] Branch foreman-packaging
-  * [ ] Create rpm/<%= release %> and update `packages/foreman/foreman-release/foreman.gpg`, `mock/*.cfg`, `package_manifest.yaml`, `rel-eng/{releasers.conf,tito.props}` and `repoclosure/*.conf`
-  * [ ] Create deb/<%= release %>
-  * [ ] Update `.github/PULL_REQUEST_TEMPLATE.md` (by adding `[ ] <%= release %>` to it) in `master`
+  * [ ] Create `rpm/<%= release %>`
+  * [ ] Create `deb/<%= release %>`
 * [ ] Bump versions to <%= develop %>-develop
```

Comment:
Correct. That's the first step in `Preparing build systems` for the Release Engineer. The actual branching is so much easier when a single person executes the steps because they're so tightly coupled that I'd even consider letting the release engineer do the actual branching of core repos as well given they're so trivial steps and easily automatable.